bitnami/mysql/templates/secondary/svc-headless.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if eq .Values.architecture "replication"}}
```

Comment:
For consistency:

```suggestion
{{- if eq .Values.architecture "replication" }}
```

Reply:
yes, in fact, this will fails. Good catch!